pkg/storaged/storage-controls.jsx

```diff
@@ -93,6 +93,8 @@ function checked(callback) {
         // only consider primary mouse button
         if (!event || event.button !== 0)
             return;
+        if (!permission.allowed)
```

Comment:
Controls can be disabled for other reasons than `!permission.allowed` so this would need to check whether the element is disabled or not.